packages/lunarclient-app/lunarclient-app.pacscript

```diff
@@ -0,0 +1,64 @@
+name="lunarclient-app"
+repology=("project: lunar-client")
+pkgver="3.1.3"
+arch=("amd64")
+pkgdesc="PvP modpack for all modern versions of Minecraft"
+homepage="https://lunarclient.com"
+depends=("libfuse2")
+replace=("lunarclient" "lunar-client")
+breaks=("lunarclient-deb" "lunarclient-bin")
+maintainer="Mythbusters123 <atr10605@icloud.com>"
+_appimage="LunarClient-${pkgver}.AppImage"
+_pkgname=lunarclient
+#_appimage="Lunar%20Client-${pkgver}.AppImage"
+# TEMPORARY ONLY UNTIL DOWNLOADING IS FIXED
+url="https://cdn.themyth.dev/${_appimage}"
+#url="https://launcherupdates.lunarclientcdn.com/${_appimage}"
+hash='f04f1364aa02f89c667f54e206326cc1f9bca8242ad2600a5cfbf748d91498b5'
+
+prepare() {
+  chmod +x "${_appimage}"
+  ./"${_appimage}" --appimage-extract
```

Comment:
Look at other appimage packages and see how they work. We try not to extract the appimages.